aiohttp/connector.py

```diff
@@ -719,6 +724,8 @@ class TCPConnector(BaseConnector):
     ttl_dns_cache - Max seconds having cached a DNS entry, None forever.
     family - socket address family
     local_addr - local tuple of (host, port) to bind socket to
+    network_interface - Name of network interface to bind socket to (e.g. "eth0").
+
```

Comment:
It doesn't seem like there's a case for having a double-empty-line separator here.
```suggestion
```